recipes/ndcube/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: ndcube
+  version: "0.1.dev273"
+
+source:
+  fn: ndcube-0.1.dev273.tar.gz
+  url: https://pypi.python.org/packages/21/e7/32f31309c75a064277babcdca88d6bb5dadefaea000303c61e177beba4d2/ndcube-0.1.dev273.tar.gz
+  md5: 36a200f32aa1ec697e1f0f0b8e6d4391
+
+build:
+  script: python setup.py install --offline --no-git --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - astropy
+
+  run:
+    - python
+    - astropy
+    - matplotlib
+    - sunpy
+
+test:
+  # Python imports
+  imports:
+    - ndcube
+    - ndcube.tests
+    - ndcube.utils
+    - ndcube.utils.tests
+    - ndcube.visualization
+
+about:
+  home: http://sunpy.org
+  license: BSD 2-Clause
```

Comment:
Please manually package the `license_file`:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

and submit an issue or PR upstream to include `LICENSE.rst` in the `MANIFEST.in` so that in future releases it is included in the source distribution. Also please put a comment here in the `meta.yaml` with the url of the issue or PR as a reminder for future updates to possibly include the license in the source.